src/gerbil/gxi.in

```diff
@@ -1,10 +1,8 @@
 #!/bin/sh
 set -e
 
-if [ -z "$GERBIL_HOME" ]; then
-    GERBIL_HOME=$(dirname $(cd ${0%/*} && echo $PWD))
-    export GERBIL_HOME
-fi
+: ${GERBIL_HOME:=@@with_prefix@@}
+export GERBIL_HOME
```

Comment:
how will this bootstrap build in stage1? build1.ss is a gerbil script!

Reply:
`GERBIL_HOME` is set by `build.sh`.  (`:=` only sets GERBIL_HOME if it was not set).

Reply:
ok, cool